### PR TITLE
Fix/thehive

### PR DESCRIPTION
**Pull request by rombernier**
Fix error with some alert with a long type like: intrusion/application-compromise
Alert type limit to 32 char in TheHive API: https://docs.strangebee.com/thehive/api-docs/#operation/Create%20Alert

Also add link to sekoia alert in description, because external link not visible on TheHive GUI

Fix and updates requested by client

### CODE REVIEW

File: TheHive/thehive/create_alert.py [hunk 1]
```diff
@@ -94,17 +94,19 @@ def run(self, arguments: dict[str, Any]):
                             )
                         )
 
+        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
+        link = f"https://app.sekoia.io/operations/alerts/{arg_alert['short_id']}"
```
Comment (Darkheir) [inline TheHive/thehive/create_alert.py:98]: Will not work with other regions than FRA1

File: TheHive/thehive/create_alert.py [hunk 2]
```diff
@@ -94,17 +94,19 @@ def run(self, arguments: dict[str, Any]):
                             )
                         )
 
+        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
```
Comment (Darkheir) [inline TheHive/thehive/create_alert.py:97]: Maybe try to cut it in a way that will be less strange in the UI ?

```suggestion
        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
        if len(alert_type) > 32:
            alert_type = arg_alert['alert_type']['category'][:32]  # limit to 32 char, max of thehive api
```

File: TheHive/thehive/create_alert.py [hunk 3]
```diff
@@ -94,17 +94,19 @@ def run(self, arguments: dict[str, Any]):
...
+        link = f"https://app.sekoia.io/operations/alerts/{arg_alert['short_id']}"
         alert: Alert = Alert(
             id=arg_alert["uuid"],
             severity=arg_alert["urgency"]["severity"] // 25 + 1,  # from 0-100 to 1-4
             date=arg_alert["created_at"],
             tags=[],
             title=arg_alert["title"],
-            type=f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}",
+            type=alert_type[:32],  # limit to 32 char, max of thehive api
```
Comment (Darkheir) [inline TheHive/thehive/create_alert.py:105]:
```suggestion
            type=alert_type
```

File: TheHive/thehive/create_alert.py (hunk 1 above)
Comment (rombernier) [reply TheHive/thehive/create_alert.py:100]: Indeed, can we keep it like that for now,
How this type of issue is managed in other module ?

Comment (Darkheir) [reply TheHive/thehive/create_alert.py:100]: Yes we can keep it this way for now. 

I think we will need to address the issue with changes in our API so the action can know in which region they are running.

**Review by Darkheir — APPROVED** [review]

**Pull request merged**